ivy_tests/test_ivy/test_frontends/test_jax/test_jax_lax_control_flow_operators.py

```diff
@@ -140,3 +140,47 @@ def _test_branch_2(x):
         branches=[_test_branch_1, _test_branch_2],
         operand=np.array(x, dtype=input_dtype),
     )
+
+
+@handle_cmd_line_args
+@given(
+    dtype_and_x=helpers.dtype_and_values(
+        available_dtypes=tuple(
+            set(ivy.valid_numeric_dtypes).intersection(
+                set(ivy_jax.valid_numeric_dtypes)
+            )
+        ),
+        min_num_dims=1,
+        min_dim_size=1,
+    ),
+    num_positional_args=helpers.num_positional_args(
+        fn_name="ivy.functional.frontends.jax.lax.while_loop"
+    ),
+)
+def test_jax_while_loop(
+    dtype_and_x,
+    num_positional_args,
+    as_variable,
+    native_array,
+    fw,
+):
+    def _test_cond_fn(x):
+        return ivy.less(x, 0, out=None)
+
+    def _test_body_fn(x):
+        return ivy.add(x, 100, out=None)
+
+    input_dtype, x = dtype_and_x
+    helpers.test_frontend_function(
+        input_dtypes=[input_dtype],
+        as_variable_flags=as_variable,
+        with_out=False,
```

Comment:
with_out should be passed in through the arguments of test_jax_while_loop as the @handle_cmd_line_args will fill in the with_out argument automatically.